system/Database/Postgre/Forge.php

```diff
@@ -130,7 +130,7 @@ protected function _alterTable(string $alterType, string $table, $field)
     protected function _processColumn(array $field): string
     {
         return $this->db->escapeIdentifiers($field['name'])
```

Comment:
This is from another PR: https://github.com/codeigniter4/CodeIgniter4/pull/6430

This was needed to get all the tests working.

Reply:
If this change from another PR, you can't include it this PR.
One PR for one thing.